scripts/notes

```diff
@@ -201,11 +201,11 @@ $notes_type Notes:
       if
         [[ -f "${__var}" ]]
       then
-        __rvm_grep 'source.*profile' "${__var}" >/dev/null 2>&1 ||
+        __rvm_grep -E '(source|\.).+profile' "${__var}" >/dev/null 2>&1 ||
         {
           important_message "
-  * WARNING: You have '~/.profile' file, you might want to load it,
-    to do that add the following line to '${__var}':
+  * WARNING: You have a '~/.profile' file, you might want to load it.
```

Comment:
it's not `a profile file`, it's `the '~/.profile' file` - at least I think so, I'm new to the distinction ;)